### PR TITLE
Add Kuberenetes tutorial content

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # text files use OS defaults on checkout, LF on checkin
 * text eol=auto
 # images are binary
-*.png binary 
\ No newline at end of file
+*.png binary 
+*.ico binary
```
Comment (rynowak) [inline .gitattributes:5]: Git was complaining about some of the icons in the repo.

File: demo/Chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+image:
+  repository: radius.azurecr.io/tutorial/webapp
+  tag: edge
```
Comment (rynowak) [inline demo/Chart/values.yaml]: @AaronCrawfis `edge` or `latest`?

Comment (AaronCrawfis) [reply demo/Chart/values.yaml]: `latest`

File: demo/Chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+image:
+  repository: radius.azurecr.io/tutorial/webapp
```
Comment (AaronCrawfis) [inline demo/Chart/values.yaml]:
```suggestion
  repository: ghcr.io/radius-project/samples/demo
```

File: demo/Chart/values.yaml (hunk 2 above)
Comment (AaronCrawfis) [reply demo/Chart/values.yaml]:
```suggestion
  tag: latest
```

**Comment by rynowak** [comment]
@aaroncrawfis - updated

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**